src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum;
+  const dayilyPay = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < 3) {
+    sum = days * dayilyPay;
+  };
+
+  if (days > 2 && days < 7) {
+    sum = days * dayilyPay - discountSmall;
+  };
```

Comment:
```suggestion
  if (days < 7) {
   return (days * dayilyPay - discountSmall);
  };
```

you may simplify your function if you will return something directly inplace